app/authenticated/project/project-catalogs/template.hbs

```diff
@@ -14,5 +14,5 @@
 
 
 <div class="footer-actions mt-20">
-  <button {{action "goBack"}} class="btn bg-transparent">{{t 'generic.back'}}</button>
+  <button {{action "goBack"}} class="btn bg-primary">{{t 'generic.back'}}</button>
```

Comment:
Using the `save-cancel` as a comparison we default cancel to `bg-transparent` I think we should stick with that.